modules/protocol/unreal4.cpp

```diff
@@ -13,14 +13,26 @@
 #include "modules/cs_mode.h"
 #include "modules/sasl.h"
 
+typedef std::map<Anope::string, Anope::string> ModData;
 static Anope::string UplinkSID;
 
+class pipesepstream : public sepstream
```

Comment:
This is only used once so you can just do 

```c++
pipesepstream sep((*moddata)["geoip"], '|');
```

... instead of creating a subclass.